app/mailers/settings_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class SettingsMailer < ApplicationMailer
```

Comment:
Since we're mailing to the user, this could have just been added to the `UserMailer`, but I like that you put in the work!